api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -49,7 +49,7 @@
 
   @VisibleForTesting
   protected BigQuery getBigQueryService() {
-    DbCdrVersion cdrVersion = CdrVersionContext.getCdrVersion();
+    DbCdrVersion cdrVersion = CdrVersionContext.nullableGetCdrVersion();
```

Comment:
This appears to be the one case where null is _expected_ - though I haven't looked into why.